web/Dockerfile

```diff
@@ -16,15 +16,15 @@ ENV REACT_APP_AUTH_REDIRECT_URI=$REDIRECT_URI
 
 WORKDIR /code
 COPY ./ ./
-RUN yarn install --immutable
+RUN yarn install --immutable --immutable-cache
 
 FROM base as development
 CMD ["yarn", "start"]
 
 FROM base as build
 RUN yarn build
 
-FROM node:12-alpine as prod
+FROM node:14-alpine as prod

```

Comment:
Any reason why we're not using `node:18-alpine`?

Reply:
no